spec/std/time/time_spec.cr

```diff
@@ -212,6 +212,14 @@ describe Time do
     time.utc?.should be_true
   end
 
+  it ".to_unix_ns" do
+    nanoseconds = 1439404155001457425
+    time = Time.unix_ns(nanoseconds)
+    time.should eq(Time.utc(2015, 8, 12, 18, 29, 15, nanosecond: 1457425))
```

Comment:
Typo?

```suggestion
    time.should eq(Time.utc(2015, 8, 12, 18, 29, 15, nanoseconds: 1457425))
```

Reply:
Unfortunately not:

https://github.com/crystal-lang/crystal/blob/d3f5d75e7d285a06f578d24659d7862d4a230b5c/src/time.cr#L464

Reply:
Indeed. Pretty inconsistent though. Anyway, thanks for clarifying! 🙏

Reply:
Ref #9002